pontoon/base/migrations/0069_notification_categories.py

```diff
@@ -0,0 +1,60 @@
+import re
+
+from django.db import migrations
+
+
+def get_category(notification):
+    verb = notification.verb
+    desc = notification.description
+
+    # New strings notifications
+    if re.match(r"updated with \d+ new string", verb):
+        return "new_string"
+
+    # Project target dates notifications
+    if re.match(r"due in \d+ days", verb):
+        return "project_deadline"
+
+    # Comments notifications
+    if re.match(r"has (pinned|added) a comment in", verb):
+        return "comment"
+
+    # New suggestions ready for review notifications
+    if verb == "":
+        return "unreviewed_suggestion"
+
+    if re.match(r"has reviewed suggestions", verb):
+        # Review actions on own suggestions notifications
+        if desc.startswith("Your suggestions have been reviewed"):
+            return "review"
+
+        # New team contributors notifications
+        if desc.startswith("<a href="):
+            return "new_contributor"
```

Comment:
Should do something explicitly (return `None` or emit error) if none of the if statements match.